tests/unittests/tests-fib/tests-fib.c

```diff
@@ -529,6 +530,32 @@ static void test_fib_14_exact_and_prefix_match(void)
     fib_deinit();
 }
 
+static void test_fib_15_get_lifetime(void)
+{
+    timex_t lifetime, now;
+    kernel_pid_t iface_id = 1;
+    char addr_dst[] = "Test address151";
+    char addr_nxt[] = "Test address152";
+    size_t add_buf_size = 16;
+    uint32_t addr_dst_flags = 0x77777777;
+    uint32_t addr_nxt_flags = 0x77777777;
+
+    TEST_ASSERT_EQUAL_INT(0, fib_add_entry(iface_id, (uint8_t *)addr_dst, add_buf_size - 1,
+                          addr_dst_flags, (uint8_t *)addr_nxt, add_buf_size - 1,
+                          addr_nxt_flags, 1000));
+
+    TEST_ASSERT_EQUAL_INT(0, fib_devel_get_lifetime(&lifetime, (uint8_t *)addr_dst, add_buf_size - 1));
+
+    /* assuming some ms passed during these operations... */
+    vtimer_now(&now);
+    timex_t cmp_lifetime = timex_add(now, timex_set(0, 900000));
```

Comment:
convinced, it implicitly tests if the conversion saving the absolute time-points works as intended.
So ACK.